pages/views/robots.njk

```diff
@@ -0,0 +1,8 @@
+---
+permalink: "/robots.txt"
+eleventyExcludeFromCollections: true
+---
+User-agent: *
+Allow: /
+
+Sitemap: {{ site.url }}/sitemap.xml
```

Comment:
It seems to me that robots.txt must contain a new line at the end of the file. (unlike the sitemap.xml file)